packages/get-platform/src/getPlatform.ts

```diff
@@ -255,7 +264,33 @@ export async function getSSLVersion(args: GetOpenSSLVersionParams): Promise<GetO
 }
 
 export async function getPlatform(): Promise<Platform> {
-  const { platform, libssl, distro, arch } = await getos()
+  const { platform, distro, arch, libssl } = await getos()
```

Comment:
Is this functionally identical to the old one?

Reply:
yes